glide.lock

```diff
@@ -1,9 +1,36 @@
 hash: 791fe92e4f8d196d1f28a3e4400108969dd94ddd2de0283916fa7c00646aa7a8
-updated: 2017-04-14T12:56:26.659073101-07:00
+updated: 2017-05-25T21:42:54.419320812-07:00
```

Comment:
yeah, I didn't pay attention to the product glide. its adding all test dependencies here. should I revert this.